pom.xml

```diff
@@ -6,6 +6,7 @@
         <groupId>org.jboss.tools</groupId>
         <artifactId>parent</artifactId>
         <version>4.4.1.Alpha1-SNAPSHOT</version>
+        <relativePath/>
```

Comment:
I don't understand how adding this with no value associated with it could be helpful. The default is "../" if none is specified.

What is the setting after this?